src/v2/Components/NavBar/menuData.ts

```diff
@@ -419,10 +232,201 @@ export const menuData: MenuData = {
+          },
+          {
+            text: "Medium",
+            menu: {
+              title: "Medium",
+              links: [
+                {
+                  text: "Painting",
+                  href: "/collection/painting",
+                },
+                {
+                  text: "Prints",
+                  href: "/collection/prints",
+                },
+                {
+                  text: "Works on Paper",
+                  href: "/collection/works-on-paper",
+                },
+                {
+                  text: "Sculpture",
+                  href: "/collection/sculpture",
+                },
+                {
+                  text: "Photography",
+                  href: "/collection/photography",
+                },
+                {
+                  text: "Textile Arts",
+                  href: "/collection/textile-art",
+                },
+                {
+                  text: "Ceramics",
+                  href: "/collection/ceramics",
+                },
+                {
+                  text: "Design",
+                  href: "/collection/design",
+                },
+              ],
+            },
+          },
+          {
+            text: "Genre",
+            menu: {
+              title: "Genre",
+              links: [
+                {
+                  text: "Abstraction",
+                  href: "/collection/abstract-art",
+                },
+                {
+                  text: "Figuration",
+                  href: "/collection/figurative-art",
+                },
+                {
+                  text: "Hyperrealism",
+                  href: "/collection/hyperrrealism",
+                },
+                {
+                  text: "Portraits",
+                  href: "/collection/portraits",
+                },
+                {
+                  text: "Landscapes",
+                  href: "/collection/landscapes",
+                },
+                {
+                  text: "Still Lifes",
+                  href: "/collection/still-lifes",
+                },
+              ],
+            },
+          },
+          {
+            text: "Rarity",
+            menu: {
+              title: "Rarity",
+              links: [
+                {
+                  text: "Unique",
+                  href: "/collection/unique-works",
+                },
+                {
+                  text: "Limited Edition",
+                  href: "/collection/limited-edition-works",
+                },
+                {
+                  text: "Open Edition",
+                  href: "/collection/open-edition-works",
+                },
+              ],
+            },
+          },
+          {
+            text: "Price",
+            menu: {
+              title: "Price",
+              links: [
+                {
+                  text: "$50,000 and Above",
+                  href: "/collect?price_range=50000-%2A",
```

Comment:
I'm sure we're missing something obvious, and i haven't given it much of a look, but query params will trigger a hard jump between pages.

Reply:
Weird. FWIW this is the same data, just re-ordered, though apparently we aren't really using all of it... which is weird on it's own.